cockatrice/src/dlg_creategame.cpp

```diff
@@ -72,12 +72,12 @@ void DlgCreateGame::sharedCtor()
     QGroupBox *joinRestrictionsGroupBox = new QGroupBox(tr("Joining restrictions"));
     joinRestrictionsGroupBox->setLayout(joinRestrictionsLayout);
 
-    spectatorsAllowedCheckBox = new QCheckBox(tr("&Spectators allowed"));
+    spectatorsAllowedCheckBox = new QCheckBox(tr("&Spectators can watch"));
     spectatorsAllowedCheckBox->setChecked(true);
     connect(spectatorsAllowedCheckBox, SIGNAL(stateChanged(int)), this, SLOT(spectatorsAllowedChanged(int)));
-    spectatorsNeedPasswordCheckBox = new QCheckBox(tr("Spectators &need a password to join"));
+    spectatorsNeedPasswordCheckBox = new QCheckBox(tr("Spectators &need a password to watch"));
     spectatorsCanTalkCheckBox = new QCheckBox(tr("Spectators can &chat"));
-    spectatorsSeeEverythingCheckBox = new QCheckBox(tr("Spectators see &everything"));
+    spectatorsSeeEverythingCheckBox = new QCheckBox(tr("Spectators can see hands"));
```

Comment:
spectatorsSeeEverythingCheckBox = new QCheckBox(tr("Spectators can see &hands"));
This will add keyboard shotcuts